projects/plugins/crm/modules/woo-sync/includes/class-woo-sync.php

```diff
@@ -228,42 +228,90 @@ public function get_settings() {
 	 */
 	public function get_woo_order_statuses() {
 		$woo_order_statuses = array(
-			'wcpending'       => __( 'Pending', 'zero-bs-crm' ),
-			'wcprocessing'    => __( 'Processing', 'zero-bs-crm' ),
-			'wconhold'        => __( 'On hold', 'zero-bs-crm' ),
-			'wccompleted'     => __( 'Completed', 'zero-bs-crm' ),
-			'wccancelled'     => __( 'Cancelled', 'zero-bs-crm' ),
-			'wcrefunded'      => __( 'Refunded', 'zero-bs-crm' ),
-			'wcfailed'        => __( 'Failed', 'zero-bs-crm' ),
-			'wccheckoutdraft' => __( 'Draft', 'zero-bs-crm' ),
+			'pending'        => __( 'Pending', 'zero-bs-crm' ),
+			'processing'     => __( 'Processing', 'zero-bs-crm' ),
+			'on-hold'        => __( 'On hold', 'zero-bs-crm' ),
+			'completed'      => __( 'Completed', 'zero-bs-crm' ),
+			'cancelled'      => __( 'Cancelled', 'zero-bs-crm' ),
+			'refunded'       => __( 'Refunded', 'zero-bs-crm' ),
+			'failed'         => __( 'Failed', 'zero-bs-crm' ),
+			'checkout-draft' => __( 'Draft', 'zero-bs-crm' ),
 		);
 		return apply_filters( 'zbs-woo-additional-status', $woo_order_statuses );
 	}
 
+	/**
+	 * Retrieve default mapped order status for a given object type
+	 *
+	 * @param int $obj_type_id Object type (e.g. ZBS_TYPE_CONTACT, ZBS_TYPE_INVOICE, or ZBS_TYPE_TRANSACTION).
+	 * @param str $order_status Woo order status.
+	 *
+	 * @return str|bool Status string to use for object
+	 */
+	public function get_default_status_for_order_obj( $obj_type_id, $order_status ) {
+		global $zbs;
+
+		$status = false;
+
+		if ( $obj_type_id === ZBS_TYPE_CONTACT ) {
+			// default contact status is configured in CRM settings
+			$status = $zbs->settings->get( 'defaultstatus' );
+		} elseif ( $obj_type_id === ZBS_TYPE_INVOICE ) {
+			// reasonable default paid mapping based on Woo status descriptions:
+			// https://woocommerce.com/document/managing-orders/#order-statuses
+			$paid_statuses = array(
+				'pending',
```

Comment:
It seems that this status is incorrect. Pending is not a paid status for WooCommerce.

> Pending payment — Order received, no payment initiated. Awaiting payment (unpaid).